zebra-rpc/src/methods.rs

```diff
@@ -362,3 +459,44 @@ pub struct GetBlock(#[serde(with = "hex")] SerializedBlock);
 ///
 /// Also see the notes for the [`Rpc::get_best_block_hash` method].
 pub struct GetBestBlockHash(#[serde(with = "hex")] block::Hash);
+
+/// Response to a `getrawtransaction` RPC request.
+///
+/// See the notes for the [`Rpc::get_raw_transaction` method].
+// pub struct GetRawTransaction(pub(super) SerializedTransaction);
+#[derive(Clone, Debug, Eq, PartialEq, serde::Serialize, serde::Deserialize)]
+#[serde(untagged)]
+pub enum GetRawTransaction {
+    /// The raw transaction, encoded as hex bytes.
+    Raw(#[serde(with = "hex")] Vec<u8>),
+    /// The transaction object.
+    Object {
+        /// The raw transaction, encoded as hex bytes.
+        #[serde(with = "hex")]
+        hex: Vec<u8>,
+        /// The height of the block that contains the transaction, or -1 if
+        /// not applicable.
+        height: i32,
```

Comment:
I can't find at https://zcash.github.io/rpc/getrawtransaction.html any field with `height` as the name. Are we sure this is what lightwalletd expects ?

Reply:
The required arguments and fields are listed in the Zebra ticket #3145, including links to the lightwalletd code. I did this for all the RPC tickets.

The line numbers are slightly different in https://github.com/adityapk00/lightwalletd , but the arguments and fields are the same.

Did you want to open a `zcashd` ticket to document the height field?

Reply:
That's true, I had to dig into the source code to find how it works, and forgot to mention that. I'll  open a ticket

Reply:
Please post the zcashd ticket here for reference after you get it open.

Reply:
It's here: https://github.com/zcash/zcash/issues/5733